cogapp/__init__.py

```diff
@@ -5,3 +5,7 @@
 """
 
 from .cogapp import Cog as Cog, CogUsageError as CogUsageError, main as main
+
+
+if __name__ == "__main__":
```

Comment:
This was here to enable calling the app as `python -m cogapp <cli_args>`. https://docs.python.org/3/library/__main__.html#idiomatic-usage

I used it while testing out the `--help` output and figured it would be useful to keep. I'm fine taking it out if you don't want it there though.